### DISCUSSION TITLE
Preference for showing Finder tabs as individual items or not

### DISCUSSION THREAD

**Discussion by jameszhan9592** [lawand-dot-io/taskbar#41] [Feature requests]
Currently, if I have one Finder window open that contains multiple Finder tabs, Taskbar shows all the tabs as separate items instead of just one Finder window. It would be nice to have the option to disable this because I often have a lot of Finder tabs open in the same Finder window, and Taskbar gets really messy real quick!

**Comment by livinlicious** [comment]
I would like this as an option as well. But dont stop the current option to show each tab as a separate task as well. Both need to be possible, im sure the community is 50/50 divided here.

**Comment by lawand-dot-io** [comment]
This feature has been implemented in Taskbar version 1.2.1 which is now released. Closing the ticket.
@jameszhan9592
